notebooks/example_end_to_end_sft_workflow.ipynb

```diff
@@ -36,13 +36,20 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "List of Sections:\n",
-    "1. Prepare Environment\n",
-    "2. Fine-Tune the Model\n",
-    "3. Load Your Fine-Tuned Model\n",
-    "4. Merge and Save the Fine-Tuned Model\n",
-    "5. Load Base Model and Fine-Tuned Model\n",
-    "6. Evaluate Model Responses Against Ground Truths"
+    "## Flow Diagram\n",
+    "\n",
+    "![SFT Trainer Image](https://raw.githubusercontent.com/glair-ai/glair-genai-doc/main/src/images/diagram/diagram_sft_trainer.png)\n",
+    "\n",
+    "The flow diagram illustrates the process of fine-tuning a Large Language Model (LLM), which includes the following steps:\n",
+    "\n",
+    "1. **Prepare Environment**: Download the necessary data and install the required libraries to begin fine-tuning.\n",
+    "2. **Fine-Tune Model**: Begin fine-tuning the Large Language Model (LLM) with specific data and configurations.\n",
+    "3. **Load Your Fine-Tuned Model**: Since fine-tuning with LoRA only provides model adapters (not the entire fine-tuned model), we need to load these adapters alongside the pretrained model.\n",
```

Comment:
How about this `Since fine-tuning with LoRA, the output only provides model adapters (not the entire fine-tuned model), we need to load these adapters alongside the pretrained model` ?

Reply:
I think the sentence before comma `Since fine-tuning with LoRA` is incomplete,
I've asked GPT to revised your suggestion and here is what I got
```
Since we are doing fine-tuning with LoRA, and the output only provides model adapters (not the entire fine-tuned model), we need to load these adapters alongside the pretrained model.
```

Reply:
Wdyt?

Reply:
Okay, I will change the suggested sentence from GPT as I stated above

Reply:
Done in latest commit

Reply:
Thank you